setup.py

```diff
@@ -9,4 +17,5 @@
       author='Yaniv Bronhaim',
       author_email='ybronhei@redhat.com',
       url="https://github.com/emesika/RaananaTiraProject",
-      scripts=["rose-client", "rose-server", "rose-admin"],)
+      scripts=["rose-client", "rose-server", "rose-admin"],
+      data_files=[('requirements.txt', [generate_requirements()]), ])
```

Comment:
This will generate the file for every setup.py run - can we do this only for sdist?